cst438-project-airline/src/main/java/cst438/service/AirlineService.java

```diff
@@ -76,6 +74,27 @@ public AirlineService(FlightRepository flightRepository, PassengerRepository pas
     return flightRepository.findAll();
   }
 
+  public Reservation makeReservation(int flightId, int userId, int seatId,
+      String passengerFirstName, String passengerLastName) {
+    ArrayList<User> user = userRepository.findByUserId(userId);
+    ArrayList<Flight> flight = flightRepository.findByFlightId(flightId);
+    ArrayList<Seat> seat = seatRepository.findBySeatId(seatId);
+
+    if (user.isEmpty() || flight.isEmpty() || seat.isEmpty()) {
+      return null;
```

Comment:
If you are returning null here, are you going to process the null value in the rest controller?